src/SprykerEco/Shared/Payone/PayoneConfig.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\Payone;
+
+use Spryker\Shared\Kernel\AbstractBundleConfig;
+
+class PayoneConfig extends AbstractBundleConfig
+{
+    public const PAYONE_BANCONTACT_AVAILABLE_COUNTRIES = [
```

Comment:
```
protected const ...

public function getPayOneBancontactAvailableCountraies()
{
   return static::
}
```

Reply:
I guess they didn't use any Config so far, since u r creating the Config only now. Why is it an issue to inject the Config?